devtools/server/actors/replay/module.js

```diff
@@ -577,6 +577,11 @@ function SendRecordingUnusable(why) {
   Services.cpmm.sendAsyncMessage("RecordingUnusable", { why });
 }
 
+function SendRecordingUnsupported(why) {
+  Services.cpmm.sendAsyncMessage("RecordingStarting");
```

Comment:
Why send this here? It should already be sent when the module itself is initialized, so double-sending it might confuse the parent process.

Reply:
The module will only send the RecordingStarting message on initialization when the process is actually recording/replaying, which it isn't if recording is unsupported.

https://github.com/RecordReplay/gecko-dev/blob/badd0c97322c4c02a3beaba9b854721301380abe/devtools/server/actors/replay/module.js#L29